projects/ngx-datatable/src/lib/components/body/body-row.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive } from '@angular/core';
+import { RowOrGroup, TreeStatus } from '../../types/public.types';
+
+@Directive({
+  selector: '[ngx-datatable-body-row]',
+  standalone: true
+})
+export class DatatableBodyRowDirective {
+  static ngTemplateContextGuard<TRow extends { treeStatus?: TreeStatus } = any>(
+    directive: DatatableBodyRowDirective,
+    context: unknown
+  ): context is {
+    row: RowOrGroup<TRow>;
+    groupValue: TRow[];
```

Comment:
Seems like this is not needed

Reply:
It is needed. its array of grouped rows which is required by body-cell component